src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectionHandler.ts

```diff
@@ -64,12 +64,20 @@ class ReconnectionProcess {
 
   async attemptPeriodicReconnection(options: ReconnectionOptions) {
     for (let i = 0; i < options.maxRetries; i++) {
-      this.reconnectDisplay.update(i + 1);
+      const currentAttempt = i + 1;
+
+      let retryInterval: number;
+      if (typeof(options.retryIntervalMilliseconds) === 'function') {
+        retryInterval = options.retryIntervalMilliseconds(currentAttempt);
```

Comment:
Could we change this to stop retrying reconnects if the `retryIntervalMilliseconds` returns null? In a world where `retryIntervalMilliseconds` can also be a `number[]` that could be converted internally to a function that returns null once you reach the end of the list instead of reusing the last value similar to SignalR's [DefaultReconnectPolicy](https://github.com/dotnet/aspnetcore/blob/86ee868589c556f63c105cb361988a0f26ffa6bc/src/SignalR/clients/ts/signalr/src/DefaultReconnectPolicy.ts).

Reply:
Yeah, that seems reasonable!